src/Control/Eff/Internal.hs

```diff
@@ -188,14 +184,13 @@ send t = E (inj t) (singleK Val)
 
 
 -- ------------------------------------------------------------------------
--- | Get the result from a pure (i.e. no effects) computation.
+-- | Get the result from a pure computation
 --
--- The type of run ensures that all effects must be handled:
--- only pure computations can be run.
+-- A pure computation has type @Eff '[] a@. The empty effect-list indicates that
+-- no furhter effect needs to be handled.
```

Comment:
typo

Reply:
I am not sure I see the same typo. Do you mean `s/effect needs/effects need/`?